### PR TITLE
Add Trader Joe on Arbitrum to dex.trades

### PR DESCRIPTION
**Pull request by pecio222**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by pecio222** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
welcome to spellbook, @pecio222 👋 

i've asked our resident wizard @Hosuke to take an initial look at your PR here

File: models/trader_joe/arbitrum/trader_joe_arbitrum_schema.yml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      - *token_sold_address
+      - *taker
+      - *maker
+      - *project_contract_address
+      - *tx_hash
+      - *tx_from
+      - *tx_to
+      - *evt_index
+
+  - name: trader_joe_trades_legacy
```
Comment (Hosuke) [inline models/trader_joe/arbitrum/trader_joe_arbitrum_schema.yml:181]: We may not need the legacy model in the current schema.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @pecio222

**Review by jeff-dude — APPROVED** [review]
thanks again for joining us in spellbook 🤝

**Pull request merged**